### DISCUSSION TITLE
Why is Table wrapped with "relative" positioning?

### DISCUSSION THREAD

**Discussion by sp88011** [shadcn-ui/ui#3088] [General]
Why does the outer div that's wrapping the `<table../>` have a relative position? What does this achieve?

[Source](https://github.com/shadcn-ui/ui/blob/3a4c3b2f7d387e4b126a8afd20badc8891a04212/apps/www/registry/default/ui/table.tsx#L4)
```
const Table = React.forwardRef<
  HTMLTableElement,
  React.HTMLAttributes<HTMLTableElement>
>(({ className, ...props }, ref) => (
  <div className="relative w-full overflow-auto">
    <table
      ref={ref}
      className={cn("w-full caption-bottom text-sm", className)}
      {...props}
    />
  </div>
))
Table.displayName = "Table"
```

**Comment by BrendanC23** [comment]
That comes from [this PR](https://github.com/shadcn-ui/ui/pull/1370).
